docs/package-lock.json

```diff
@@ -1,4353 +1,8 @@
 {
   "name": "fluidframework-docs",
   "version": "0.25.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
If it's a hassle to deal with different versions, please go ahead and commit this to revert back to v1 lockfiles. I didn't consider the annoyance of applying dependency upgrades across the repo when the lockfile formats are different. I wouldn't have made the change originally if I had considered that.